docs/source/models/lora.rst

```diff
@@ -107,3 +107,55 @@ The following is an example request
             "max_tokens": 7,
             "temperature": 0
         }' | jq
+
+
+Dynamically serving LoRA Adapters
+---------------------------------
+
+In addition to serving LoRA adapters at server startup, the vLLM server now supports dynamically loading and unloading
+LoRA adapters at runtime through dedicated API endpoints.This feature can be particularly useful when the flexibility
```

Comment:
LoRA adapters at runtime through dedicated API endpoints.This feature can be particularly useful when the flexibility
```suggestion
LoRA adapters at runtime through dedicated API endpoints. This feature can be particularly useful when the flexibility
```